### PR TITLE
Show a nice folder structure in Visual Studio

### PR DESCRIPTION
**Pull request by mrexodia**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://cutter.re/docs/contributing/code/getting-started.html) to this repository
- [x] I made sure to follow the project's [coding style](https://cutter.re/docs/contributing/code/development-guidelines.html)
- [x] I've updated the [documentation](https://cutter.re/docs/user-docs.html) with the relevant information (if needed)

Just makes Visual Studio look nice:

image

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -440,8 +440,11 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU"
     set_source_files_properties(${BINDINGS_SOURCE} PROPERTIES COMPILE_FLAGS -w)
 endif()
 
+# Make a source group for Visual Studio
+set(ALL_SOURCES ${OPTIONS} ${UI_FILES} ${QRC_FILES} ${PLATFORM_RESOURCES} ${SOURCES} ${HEADER_FILES} ${BINDINGS_SOURCE})
+source_group(TREE "${CMAKE_CURRENT_LIST_DIR}" FILES ${ALL_SOURCES})
 
-add_executable(Cutter ${OPTIONS} ${UI_FILES} ${QRC_FILES} ${PLATFORM_RESOURCES} ${SOURCES} ${HEADER_FILES} ${BINDINGS_SOURCE})
+add_executable(Cutter ${ALL_SOURCES})
```
Comment (thestr4ng3r) [inline src/CMakeLists.txt]: I think you will need to remove `${BINDINGS_SOURCE}` above and re-add it here to fix this:
```
CMake Error at src/CMakeLists.txt:445 (source_group):
  source_group ROOT: /home/runner/work/cutter/cutter/src is not a prefix of
  file:
  /home/runner/work/cutter/cutter/build/src/bindings/CutterBindings/cutterbindings_module_wrapper.cpp
```

**Review by thestr4ng3r — CHANGES_REQUESTED** [review]
Sorry for the delay. The overall approach looks good to me.

File: src/CMakeLists.txt (hunk 1 above)
Comment (ITAYC0HEN) [reply src/CMakeLists.txt]: ping

Comment (mrexodia) [reply src/CMakeLists.txt]: Sorry for the late reply, should be updated now.

**Review by thestr4ng3r — APPROVED** [review]

**Pull request merged**